versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/azure-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Installing Rancher Prime PAYG on Azure
+---
+
+This page covers installing the Rancher Prime PAYG offering on Microsoft's Azure Marketplace.
+
+## How to Install Rancher Prime PAYG
+
+The following steps describe how to create a new deployment of Rancher Prime from the [Azure Marketplace](https://azuremarketplace.microsoft.com) page.
```

Comment:
The occurrence on the Azure page under pages-for-subheader should have the link since that's the first occurrence of Azure Marketplace. Depending on how we feel about multiple links to the same target, the link here can be removed.

Same goes for AWS.

Reply:
Got it, removed the link on this page and added the link to the pages-for-subheader per f0d4deec5d747abcecbb327c73144b45f7904eaa